love/src/components/ScriptQueue/Scripts/FinishedScript/FinishedScript.jsx

```diff
@@ -202,32 +192,36 @@ export default class FinishedScript extends PureComponent {
             className={[scriptStyles.expandedSectionWrapper, this.state.expanded ? '' : scriptStyles.hidden].join(' ')}
           >
             <ScriptDetails {...this.props} />
-            <div className={styles.configurationContainer}>
-              <div>
-                APPLIED CONFIGURATION
-                {this.state.coppiedToClipboard && <div className={styles.fadeElement}>Coppied to clipboard</div>}
-                {/* <div className={styles.fadeElement}>Coppied to clipboard</div> */}
-                <div className={styles.copyToClipboardIcon} onClick={this.copyConfigToClipboard}>
-                  <CopyIcon title="Copy to clipboard" />
+            {this.state.appliedConfiguration && (
+              <div className={styles.configurationContainer}>
+                <div>
+                  APPLIED CONFIGURATION
+                  {this.state.coppiedToClipboard && <div className={styles.fadeElement}>Coppied to clipboard</div>}
+                  {/* <div className={styles.fadeElement}>Coppied to clipboard</div> */}
+                  <div className={styles.copyToClipboardIcon} onClick={this.copyConfigToClipboard}>
+                    <CopyIcon title="Copy to clipboard" />
+                  </div>
                 </div>
+                <AceEditor
+                  mode="yaml"
+                  theme="solarized_dark"
+                  name="UNIQUE_ID_OF_DIV"
+                  width={'26em'}
+                  height={'150px'}
+                  value={this.state.appliedConfiguration}
+                  // editorProps={{ $blockScrolling: true }}
```

Comment:
If this will not be used in the future, it should be removed.

Reply:
Good catches @areyesd14 !! thanks :raised_hands: